src/bidiMapper/modules/network/CollectorsStorage.ts

```diff
@@ -138,36 +160,60 @@ export class CollectorsStorage {
       // Collector is aimed for a different top-level browsing context.
       return false;
     }
-    if (collector.maxEncodedDataSize < request.bytesReceived) {
+    if (!collector.dataTypes.includes(dataType)) {
```

Comment:
I guess the current implementation is to have `dataTypes` as Array?
Does a change to Set make sense there?

Reply:
I see, interesting I didn't know about the performance parts